Core/F0AM_ModelCore.m

```diff
@@ -339,6 +446,11 @@
 %Chemical rates
 Conc(:,2) = sum(Conc(:,Chem.iRO2),2);
 G = Conc(:,Chem.iG(:,1)).*Conc(:,Chem.iG(:,2));
+%Need to adjust G here as well to deal with limiting reactants in heterogeneous
```

Comment:
This section fixes the output values of Rates to be the correct value for reactions with limiting reagents.